models/DataObject/Service.php

```diff
@@ -66,10 +66,10 @@ class Service extends Model\Element\Service
      *
      * @var string[]
      */
-    private const BC_VERSION_DEPENDENT_DATABASE_COLUMNS = ['id', 'parentId', 'type', 'key', 'path', 'index', 'published',
-                                                                'creationDate', 'modificationDate', 'userOwner', 'userModification',
-                                                                'classId', 'childrenSortBy', 'className', 'childrenSortOrder',
-                                                                'versionCount', ];
+    private const BC_VERSION_DEPENDENT_DATABASE_COLUMNS = ['id', 'parentid', 'type', 'key', 'path', 'index', 'published',
+                                                                'creationdate', 'modificationdate', 'userowner', 'usermodification',
+                                                                'classid', 'childrensortby', 'classname', 'childrensortorder',
+                                                                'versioncount', ];
```

Comment:
```suggestion
    private const BC_VERSION_DEPENDENT_DATABASE_COLUMNS = ['id', 'parentid', 'type', 'key', 'path', 'index', 'published',
                                                                'creationdate', 'modificationdate', 'userowner', 'usermodification',
                                                                'classid', 'childrensortby', 'classname', 'childrensortorder',
                                                                'versioncount'];
```

is that last comma intended?